### DISCUSSION TITLE
Missing documentation for Allure plugins

### DISCUSSION THREAD

**Discussion by fkorsa** [allure-framework/allure2#2167] [Documentation] [answered]
There used to be a section in the documentation describing how to create allure plugins. I can't seem to find it anymore.

Did it move somewhere else, or was it completely removed? Without an official how-to, it is very difficult to start writing a new plugin from scratch.

**Comment by eroshenkoam** [comment, marked as answer]
We've updated our documentation, which can now be found at:
https://allurereport.org/docs

However, it doesn't cover information about plugins.

For that, you can refer to our GitHub repository:
https://github.com/allure-framework/allure-docs/blob/main/content/plugins/overview.md

**Reply by fkorsa** [reply]
Thank you, that's perfect!

Maybe there could be a link to this page from the main documentation?
